CommunityFeatures.md

```diff
@@ -1,40 +1,76 @@
-# Introduction
-
-Every time a Pull request improves the community firmware it shall note down it's achievements and usage in this document.
-
-
-# General improvements
-
-Here is a list of general improvements that have been made ordered from newest to oldest:
-
-* PR #1 - This list was started and it's first entry shall be replaced with the first improvement
-
-
-# Added features
-
-Here is a list of features that have been added to the firmware as a list ordered from newest to oldest:
-
-* PR #7 - This list was started and it's first entry shall be replaced with the first improvement
-
-    A documentation of the feature, how it can be used and configured shall be put in this position below every entry 
-
-
-# Runtime settings aka Community Features Menu
-
-In the main menu of the deluge (Shift + Pressing selection knob) there is an entry called "Community Features" that allows changing behavior and turning features on and off in comprison to the original and previous community firmwares. Here is a list of all options and what they do:
-
-* Force 7SEG Mode
-
-    Enable or disable, shows 7SEG rendering on the OLED instead of the OLED screen. Mainly intended for debugging purposes. Currently a dream and not really implemented
-
-# Compiletime settings
-
-This list includes all preprocessor switches that can alter firmware behaviour at compile time and thus require a different firmware
-
-* HAVE_OLED
-
-    Currently determines if the built firmware is intended for OLED or 7SEG hardware
-
-* FEATURE_...
-
-    Description of said feature, first new feature please replace this
\ No newline at end of file
+# Introduction
+
+Every time a Pull request improves the community firmware it shall note down it's achievements and usage in this document.
+
+# File Compatibility Warning
+In general, we try to maintain file compatibility with the official firmware. However, **files (including songs, presets, etc.) that use community features may not ever load correctly on the official firmware again**. Make sure to back up your SD card!
+
+# General improvements
+
+Here is a list of general improvements that have been made ordered from newest to oldest:
+
+* PR [#17] - Increase the resolution of "patch cables" between mod sources and destinations.
+* PR [#29] - Distinguish between MPE and non-MPE kit rows.
+* PR [#47] - Extra MIDI ports on the USB interface for MPE. This improves the usability of MPE-capable devices through the USB interface by providing more virtual cables.
```

Comment:
@m-m-adams I think this is correct, but please double check the description for #47 and #49.